doc/source/operations.rst

```diff
@@ -234,6 +234,23 @@ following information to the "osh" section under "charts":
    such as Deckhand and Shipyard belong under "ucp", OpenStack services belong
    under "osh", and infrastructure components belong under "osh_infra".
 
+Reboot Compute Host
+===================
+
+Before reboot compute host, Shutdown all VM(s) from that compute host.
+
+After reboot the compute host, It is possible that the pods when started come up out of order.
+If this happens, you might see symptoms of the vm not getting an ip address.
+To address this problem, run the following commands
+
+.. code-block:: console
+
+   kubectl get pods -o wide | grep ovs-agent | grep <compute name>"
+   kubectl delete pod -n openstack <ovs-agent pod name>
+
+This should restart the ovs-agent pod and reconfigure the vxlan tunnel network configuration.
```

Comment:
@sgrasley @gracetai5 did we have to stop/start the Nova servers?

Reply:
We did not need to as part of troubleshooting I was involved in.